infrastructure/cdn-in-a-box/docker-compose.yml

```diff
@@ -209,48 +209,36 @@ services:
       - variables.env
     hostname: trafficvault
 
-  # Apache Traffic Server (ATS) caches defined here
-  # base image from which all other caches inherit (builds and installs ATS+plugins)
-  tccache:
-    build:
-      context: ../..
-      dockerfile: infrastructure/cdn-in-a-box/cache/Dockerfile
-    image: tccache:latest
-
   # edge cache
   edge:
     privileged: True
     build:
-      context: .
-      dockerfile: edge/Dockerfile
-    depends_on:
-      - enroller
```

Comment:
Shouldn't this still depend on the enroller?

Reply:
It's not necessary. Dependency is used to guarantee the startup sequence of container, not the services inside.
You can take a look at:
`infrastructure/cdn-in-a-box/dns/set-dns.sh`: waiting for dns
`to-enroll` of `infrastructure/cdn-in-a-box/traffic_ops/to-access.sh`: waiting for the services enrollment

Reply:
I guess with the cert stuff there's enough going on at the beginning that you don't notice much that the enroller isn't up yet.

Reply:
Both edge and mid wait the cert stuff in `run.sh`
[edge/run.sh](https://github.com/apache/trafficcontrol/blob/master/infrastructure/cdn-in-a-box/edge/run.sh#L30)
[mid/run.sh](https://github.com/apache/trafficcontrol/blob/master/infrastructure/cdn-in-a-box/mid/run.sh#L30)

Reply:
yeah, exactly.